source/configuring_domain/02_managing_legal_information/legals.rst

```diff
@@ -68,22 +68,68 @@ To specify the Terms and Conditions and the Privacy Policy of a portal, go to Co
 Licenses
 --------
 
-When configuring the metadata of a dataset in the Information tab, the license attached to that dataset should be defined, using a drop-down selection of predefined licenses. To define the licenses listed in the drop-down selection, go to Configuration > Legals. In this subsection of the back office menu, look for the "Licenses" area.
+When configuring the metadata for a dataset in the Information tab, it is important to define the license that specifies how others can use the data.
 
-.. image:: images/license_configuration.png
+Configure licenses
+~~~~~~~~~~~~~~~~~~
 
-1. Click the Add license button.
-2. In the Labels column, write the label of the license. The label will be displayed both in the drop-down selection in the back office, and in the Information tab of the published dataset in the front office. There must be one label per language available on the domain.
-3. (optional) In the URLs column, indicate the official website of the defined license, in order to find more information about that license. There can be one URL per language available on the domain.
-4. Click the Save button in the top right corner of the page.
+In the back office, you can configure the licenses available when adding the metadata for a dataset.
+
+1. From the left menu under Configuration, select Legals.
+2. Go to the Licenses area.
+   
+   .. image:: images/license_configuration.png
+
+3. Click the Add license button.
+4. (optional) In the URLs column, add the URL to the official website of the defined license to find more information about that license. You can only add one URL per language available on the domain.
```

Comment:
(suggestion) insist that providing the official, "legal" definition of the license is a good practice (in order to make the data easily re-usable), + in the case of "custom licenses" (ie. the customer has their own licenses overseen by their legal department), it can be a page of a corporate website, or a page done/hosted on their ODS platform

Reply:
Sure, I'll add a note here about this.